cmd/helm/package.go

```diff
@@ -166,7 +166,7 @@ func (p *packageCmd) clearsign(filename string) error {
 // promptUser implements provenance.PassphraseFetcher
 func promptUser(name string) ([]byte, error) {
 	fmt.Printf("Password for key %q >  ", name)
-	pw, err := terminal.ReadPassword(int(syscall.Stdin))
+	pw, err := terminal.ReadPassword(syscall.Stdin)
```

Comment:
did you manually test this one? I seem to recall that I had to assert int for some reason here.